### PR TITLE
Ms tabbed interface

### PR DESCRIPTION
**Pull request by MayaS1111**
Created routes, controller methods, and pages for feed, followers, and following using partials

### CODE REVIEW

File: app/views/users/feed.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<h1>
+  <%= @user.username %>
+</h1>
+
+<div class="row">
```
Comment (jb520) [inline app/views/users/feed.html.erb]: Everything looks good! I ended up making a partial for the user navbar to save some space and so I'd only have to write out the links once, if you have some spare time I'd suggest doing that!

**Review by jb520 — APPROVED** [review]
Good job!

File: app/views/users/feed.html.erb (hunk 1 above)
Comment (MayaS1111) [reply app/views/users/feed.html.erb]: Thanks, I saw that on yours; super smart!

**Review by CarlosCastanedaDev — APPROVED** [review]
Nice job creating the navbar partial. You can now merge these changes back to main.

**Review by kcalva — APPROVED** [review]
Everything looks good. The only suggestion i have (if you have time) is that this line 
`<h1><%= @user.username %></h1>`
seems to be repeated on each of your view files. I suggest you add that to the partial (_profilenav.html.erb)

File: app/controllers/users_controller.rb
Comment (SaraDawner2000) [inline app/controllers/users_controller.rb:1]: You could DRY the code up a little by putting the user fetching into a private method and adding a before_action that calls it

**Comment by MayaS1111** [comment]
thanks for catching that i forgot to move it

File: app/views/follow_requests/_follow_request.html.erb
Comment (SaraDawner2000) [inline app/views/follow_requests/_follow_request.html.erb:1]: Was this in the lesson?

File: app/views/users/_profilenav.html.erb
Comment (SaraDawner2000) [inline app/views/users/_profilenav.html.erb:1]: Great job abstracting this into a partial!

File: app/views/likes/_like.html.erb
Comment (SaraDawner2000) [inline app/views/likes/_like.html.erb:1]: is this neccessary?

**Review by SaraDawner2000 — APPROVED** [review]
Looks great!

File: app/views/users/followers.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+<%= render "users/profilenav", user: @user%>
+
+<h2>Followers</h2>
+<ul>
```
Comment (SaraDawner2000) [inline app/views/users/followers.html.erb:5]: Maybe this could user some bootstrap styling? (list group)

**Pull request merged**